public/shims/processing/p5/p5-shim.js

```diff
@@ -23,13 +23,13 @@ const $builtinmodule = function (name) {
   let vectorClass;
   let fontClass;
 
-  const mod = {__name__: new Sk.builtin.str("p5")};
+  const mod = { __name__: new Sk.builtin.str("p5") };
 
   // Helper function for extracting values from arguments
   const processArgs = function processArgumentValues(arguments_) {
```

Comment:
could this use arrow function syntax?

`const processArgs = (arguments_)  => { ... } `

Reply:
Maybe, although tbh it might be worth going through the whole file at some point and transferring to ES6 syntax if it works, because this is literally everywhere 😅